install/kubernetes/cilium/values.yaml

```diff
@@ -93,6 +93,14 @@ image:
 
 # -- Affinity for cilium-agent.
 affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: kubernetes.io/os
+              operator: In
+              values:
+                - linux
```

Comment:
I didn't want to do a bunch of time travel on a fast-moving codebase and think the existing `CODEOWNERS` may have a better perspective. I suspect the removal of Linux was done to support the non-amd64 arch and someone else duplicated the small work in this PR to remove the deprecation.

If a code owner or maintainer concurs, please feel free to close this PR.
Cheers,
-steve

Reply:
I agree with the change, but please change the commit description and PR title to reflect that you are re-introducing the nodeAffinity selector. The PR description (which will be used for release notes) and the commit message still mention beta annotation, which is not the case anymore.